src/fmu/dataio/datastructure/export/content.py

```diff
@@ -71,3 +71,39 @@ class AllowedContent(BaseModel):
     relperm: None = Field(default=None)
     lift_curves: None = Field(default=None)
     transmissibilities: None = Field(default=None)
+
+    @staticmethod
+    def requires_additional_input(field: str) -> bool:
+        # Ideally 'property' and 'seismic' should have been part of the below
+        # filds that requires additional input, but due to backwards compatibility
+        # they have to be excluted (for now).
+        if field in ("property", "seismic"):
+            warnings.warn(
+                f"In future versions '{field}' must be given as an composite: "
```

Comment:
I don't think the term "composite" means anything to the average user. Warning text should prompt user to change _now_, as this will be disallowed in the future - and it should give some hints as to what to change to.

(Not a particularly great) suggestion:

```
When using content "property", please use a dictionary form, as more information is required. Example: content={"property": {"is_discrete": False}}

The use of {field} will be disallowed in future versions."
```

(would have to be a different example for `seismic` obviously)